aaudio/README.md

```diff
@@ -1,15 +1,40 @@
 AAudio Samples
 ==============
 These samples demonstrate AAudio API:
-1. hello-aaudio sample: creates output stream
-1. echo sample: creates output and input streams, directly play captured audio
+1. hello-aaudio sample: creates an output stream
+1. echo sample: creates output and input streams, then play via loopback
 
-Requirements
+Pre-requisites
 -----------
-* Android Device with Android API 25 ( Andorid O )
-* A headphone( or a speaker ) plugged into Android Device
-* Support Audio Recording
-* Android Studio 2.3.0+
+* Android Device with recording capability, running Android API 25( Android O )
+* An external headphone ( with a microphone ) plugged into Android Device
```

Comment:
Nit: Don't put spaces inside of parenthesis. "(with a microphone)", not "( with a microphone )".